newIDE/app/src/UI/SearchBar.js

```diff
@@ -29,6 +29,8 @@ type Props = {|
   buildTagsMenuTemplate?: () => any,
   /** If defined, a help icon button redirecting to this page will be shown */
   helpPagePath?: ?string,
+  /** If defined, the tabIndex value is applied to all buttons. Useful for taking them out of the focus order when desired via -1 */
+  buttonsTabIndex?: string,
```

Comment:
Why taking buttons out of the focus? This can reduce accessibility for people for which using a mouse is difficult. I don't mind 1 or 2 extra key press on the Tab key. In theory there is already a callback on the SearchBar that is set so that when you search something and press enter, the first result is selected (at least for the object/actions/conditions selector)

Reply:
I want this workflow to work quick, so when you search, tabbing takes you straght to the list items and lets you select one in the result then hit enter. I hind it much less likely that I will want to tab to any buttons. I want tabbing to take me to the results, If I am tabbing it is more likely that I saw something in the list I want to select :)

Reply:
Sure but in this case I would prefer to have Up/Down buttons allowing you to change the selection and "Enter" choose the currently selected object or action/condition (a bit like in text editors like when doing Ctrl+P in VSCode).

Tab should still allow to iterate on all buttons - which is the default thing in all dialogs and which allow GDevelop dialogs to be almost always entirely usable with the keyboard.

But in any case, let's do these improvements on a separate PR - I would strongly recommend to go the up/down button + enter way as it seems more standard (a bit like a dropdown)

Reply:
How about removing them from focus order only when the user has entered something in the search box? It will also simplify it so less props to pass. I wanted to have it tab so it can be used with the left hand :) the up/down approach is going to be hard, since the list component alr ady has that mapped to scrolling

Reply:
Btw hitting enter already selects the object. You can type to s arch, tab to an object and hit enter, then search straight away for action or condition, tab to it again and hit enter again.

Reply:
Ok I am removing this now. Maybe another PR :)